src/api/event/routes/event.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+/**
+ * event router
+ */
+
+const { createCoreRouter } = require('@strapi/strapi').factories;
```

Comment:
In order to satisfy `Events endpoints should not be available through API.` should we disable routes or controller?

Reply:
I think that's a good idea. Will swagger still try to create documentation thinking the API is available?

Reply:
Let's keep API for now. We will need to make a decision  for both Events and Subscriptions. Probably the easiest approach would be to disable any permissions on API for all roles in deployment but enable readOnly for tests. But also we can drop API routes, though we will need to use internal Services in Strapi within tests.